src/components/BorrowerProfile/LoanUse.vue

```diff
@@ -1,10 +1,19 @@
 <template>
 	<p>
 		{{ loanUseFiltered }}
+		<kv-text-link
+			v-if="loanId"
+			:to="`/lend/${loanId}`"
+			v-kv-track-event="['Lending', 'click-Read more', 'Learn more', loanId]"
+		>
+			Learn more
+		</kv-text-link>
 	</p>
 </template>
 
 <script>
+import KvTextLink from '~/@kiva/kv-components/vue/KvTextLink';
+
 const loanUseFilter = require('../../plugins/loan-use-filter');
```

Comment:
Unrelated to this PR, but can this `require` statement be removed in favor of a standard `import`?